lib/pseudos.js

```diff
@@ -189,6 +189,13 @@ function getFirstElement(elems, adapter) {
     }
 }
 
+//pseudos that depends on optional Adapter methods. (key, value) = (pseudoName, adapterMethod)
+var pseudosWithOptionalImplementation = {
```

Comment:
I kept this list, so that before getting function for particular `pseudo`, we can check if its associated Adapter method is implemented or not. Otherwise we will simply use `falseFunc`.